ci/bugfix-release-pr-body.md

```diff
@@ -0,0 +1,52 @@
+AUTO: merge $RELEASE_BRANCH into $SOURCE_BRANCH
+
+This PR was automatically generated in response to a push to `master`,
+and is a chance to review any changes that will be included in the release
+branch before merging.  Under most circumstances, this PR will probably be
+a formality.  However, there are a few cases where we may need to be some
```

Comment:
"may need to **do** some extra work."  

Typo here, and since we're going to be seeing this text over and over....

Reply:
Ah, thanks for catching this!  I've updated the text in 511c395.